.github/workflows/ci.yml

```diff
@@ -296,3 +296,28 @@ jobs:
         run: cargo fmt --all --manifest-path=esp-riscv-rt/Cargo.toml -- --check
       - name: rustfmt (examples)
         run: cargo fmt --all --manifest-path=examples/Cargo.toml -- --check
+
+  build-docs:
```

Comment:
This should not be running for every commit pushed to every PR. We should either create a proper release workflow, or just create a new documentation workflow which is triggered by the `workflow_dispatch` event for now, so we can manually invoke it.